vpp.env

```diff
@@ -1,5 +1,5 @@
 VPP_REPO_URL=https://github.com/FDio/vpp.git
-VPP_DEFAULT=1908
+VPP_DEFAULT=2001
```

Comment:
Why are we switching the default? The VPP 20.01 is still in development and has not been released yet.